src/slic3r/GUI/wxExtensions.hpp

```diff
@@ -18,14 +18,17 @@
 
 #include "Widgets/PopupWindow.hpp"
 
-#ifdef __WXMSW__
-void                msw_rescale_menu(wxMenu* menu);
-#else /* __WXMSW__ */
-inline void         msw_rescale_menu(wxMenu* /* menu */) {}
-#endif /* __WXMSW__ */
+#ifndef __linux__
+void sys_color_changed_menu(wxMenu* menu);
+#else
+inline void sys_color_changed_menu(wxMenu* /* menu */) {}
+#endif // no __linux__
+
+[[deprecated("Function renamed to sys_color_changed_menu. This function now redirects to sys_color_changed_menu")]]
+inline void msw_rescale_menu(wxMenu* menu) { sys_color_changed_menu(menu); }
```

Comment:
Depreciated function with old name. Also done to other msw_rescale functions that were renamed. Should these be kept?

Reply:
This is very nice!